lib/library.cpp

```diff
@@ -1361,3 +1361,23 @@ bool Library::isSmartPointer(const Token *tok) const
     return smartPointers.find(typestr) != smartPointers.end();
 }
 
+const Library::Container * getLibraryContainer(const Token * tok)
```

Comment:
```suggestion
CPPCHECKLIB const Library::Container * getLibraryContainer(const Token * tok)
```
`CPPCHECKLIB` must be prepended here also